pkg/apiserver/registry/anomalydetector/throughputanomalydetector/rest.go

```diff
@@ -0,0 +1,212 @@
+// Copyright 2022 Antrea Authors
```

Comment:
The the new files should have 2023 instead of 2022
```suggestion
// Copyright 2023 Antrea Authors
```